components/camel-casper/src/main/docs/casper-component.adoc

```diff
@@ -0,0 +1,64 @@
+= Casper Camel Connector Component
+:doctitle: Casper Camel Connector
+:shortname: casper
+:artifactid: camel-casper
+:description: Camel casper endpoint : to interract with Casper nodes
```

Comment:
I'd follow the description pattern used in other components. Please look at how we write the description for the `camel-kafka` component and try to describe this one on the same way.